cart.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Don't forget to change the document title since it's very important for search engine optimization (SEO)! The page title is used by search engine algorithms to decide the order when listing pages in search results.

check this out: [HTML <title> tag](https://www.w3schools.com/TAGs/tag_title.asp)